utils.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+
+def get_rows_and_columns():
+    while True:
+        try:
+            rows = int(input('Enter the number of rows: '))
+            columns = int(input('Enter the number of columns: '))
+            return rows, columns
+        except ValueError:
+            print("Please enter valid integer values for rows and columns.")
```

Comment:
now you have function to grab two int values from user. and what if you need 3 either 1or 7 values. are you going to write different functions for every amount of variables